packages/web/src/components/App.tsx

```diff
@@ -38,8 +39,22 @@ export function App({ locationChangeHandler }: AppProps) {
         <meta property='og:image' content='/static/sharing-image.png' />
       </Helmet>
       <Routes>
-        <Route index element={<HomePage />} />
-        <Route path='/scan/*' element={<WebsiteResultsPage />} />
+        <Route
+          index
+          element={
+            <RollbarContext context='/'>
```

Comment:
Can you use a single context per App component?

Reply:
This is intended to be used on the routes layer to make logs more specific, and it's not mandatory at all. I thought it would be better to have info about what route the error happened at.

Reply:
Can't see why would we need this. Errors (should) clearly map to sources by stacktraces already.

Reply:
Ok, removed it